tests/sitemaps_tests/test_generic.py

```diff
@@ -57,3 +58,21 @@ def test_generic_sitemap_lastmod(self):
 """ % (self.base_url, test_model.pk)
         self.assertXMLEqual(response.content.decode(), expected_content)
         self.assertEqual(response.headers['Last-Modified'], 'Wed, 13 Mar 2013 10:00:00 GMT')
+
+    def test_get_protocol_defined_in_constructor(self):
+        sitemap = GenericSitemap({"queryset": None}, protocol='http')
+        self.assertEqual(sitemap.get_protocol(), 'http')
+
+    def test_get_protocol_passed_as_argument(self):
+        sitemap = GenericSitemap({"queryset": None})
+        self.assertEqual(sitemap.get_protocol('http'), 'http')
```

Comment:
Are these tests not already covered elsewhere? If not, then they should be added first as a separate commit as they are unrelated to this deprecation.

Reply:
`grep -r "get_protocol" tests/` does not return any other results. I've separated the commits now.